src/bindings/swig/lua/tests/test_gopts_kdb.lua

```diff
@@ -0,0 +1,87 @@
+--[[
+-- TODO: cannot call std::vector<std::string> function from lua
```

Comment:
The `std::vector<std::string>` solution I used for Ruby and Python doesn't work for Lua (it seems Lua Tables can't easily be converted into `std::vector`). Do we have someone who knows more about Lua and SWIG, or should we just leave `elektraGOptsContract` unsupported in Lua?

Reply:
@manuelm as you are our best Lua hacker, do you have some idea how this can be done?

Reply:
This should work out of the box:
```lua
local custom_argv = kdb.StringVector()
custom_argv:push_back("test")
custom_argv:push_back("get")
custom_argv:push_back("-v")
custom_argv:push_back("user:/")
```

Reply:
I already found a different solution, but thanks for the help